app/src/main/java/com/codesoom/assignment/application/ProductService.java

```diff
@@ -0,0 +1,89 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.entity.Product;
+import com.codesoom.assignment.exception.ProductNotFoundException;
+import com.codesoom.assignment.repository.ProductRepository;
+
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class ProductService {
+
+    private final ProductRepository productRepository;
+
+    public ProductService(ProductRepository productRepository) {
+        this.productRepository = productRepository;
+    }
+
+    /**
+     * 새로운 상품을 생성한다.
+     *
+     * @param product the product
+     * @return 저장한 상품
+     */
+    public Product create(Product product) {
+       return productRepository.save(product);
+    }
+
+    /**
+     * id로 product 객체를 찾아 리턴한다
+     *
+     * @param id the id
+     * @return 찾은 상품
+     */
+    public Product findById(Long id) {
+        return productRepository.findById(id).orElseThrow(() -> new ProductNotFoundException("해당 id를 가진 상품이 존재하지 않습니다 id:" + id));
```

Comment:
한 줄이 너무 길어지지 않도록 합시다.
```suggestion
        return productRepository.findById(id)
            .orElseThrow(() -> new ProductNotFoundException("해당 id를 가진 상품이 존재하지 않습니다 id:" + id));
```